imageresizerapp/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,7 @@
+﻿Install-ChocolateyPackage `
+    -PackageName 'imageresizerapp' `
+    -FileType 'EXE' `
+    -SilentArgs '/quiet /norestart' `
+    -Url 'https://github.com/bricelam/ImageResizer/releases/download/v3.0/ImageResizerSetup-3.0.exe' `
```

Comment:
Yeah, this is what the docs were recommending. The only thing that is different is whether the shell extension shows up in both 64-bit and 32-bit processes or just 32-bit processes. Using `-forcex86` (the way it's intend) would be shooting yourself in the foot.